conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,265 @@
+  * **Headline:** A headline is usually phrased as a question. Include important keywords (like “permanent”) in the headline. Avoid extraneous copy, like *Are you sure?*, which distracts users from the main outcome of the action.
+  * **Body text:** Body text gives information about the action’s consequence.
+  * **Buttons:** Buttons allow a user to answer the headline question. These can simply be “yes” or “no,” but adding some context increases clarity—especially when skimming.
+* For error messages, use [PatternFly’s error messaging guidelines](https://www.patternfly.org/v4/ux-writing/error-messages).
+
+### Check your work module
+Once a user completes a step, a Check your work module appears. This module prompts the user to answer a yes/no question about the step results, which gives them the opportunity to review their work. For this module, you’ll only need to write a single yes/no question.
+
+If the user answers Yes, a check mark will appear.
+
+If the user answers No, an error message appears with a link to relevant documentation, if necessary. The user then has the opportunity to go back and try again.
+
+
+## <a name="partiv"></a> Part IV: How can I contribute a quick start?
+
+### Internal contributions
+
+#### Process goals:
+* To provide user-centric, functional quick starts for OpenShift users across a wide range of product areas
+* To create a successful collaboration across teams (UXD, CCS, PM, etc.) for every quick start
+* To define which teams are involved at each step of the quick start effort so that the end result is organized, clean, and in final form for users
+* To respect the bandwidth and input of each team
+
+#### Phase 1: Planning
+1. Determine which product area needs a quick start.
+  * Identify stumbling blocks or complex areas where we can enable user success with as little friction as possible. This information will be gathered from:
+    * UXD user research
+    * Pain points shared from internal feedback sessions (Solution Architects, Technical Marketing, CEE/Customer Support, etc.)
+    * Product Management input
+    * OpenShift or Kubernetes documentation
+    * Features we want to highlight or push more to entice users to try
+    * Areas we want to offer more opinionated views or flows. For example, there are likely cases where there are multiple ways for users to accomplish a task, but we’re recommending a method or suggesting how to get up and running with XYZ.
+  * Note that regardless of where the quick start idea originates, quick starts should always be a part of an epic.
+  * **Team responsible:** Product Management; UXD Design; possibly UXD Research
+  * **Time frame:** Before epic prioritization
+  * **Definition of done:** Product Management, UXD Design, UXD Research, and any other groups in this step agree on the product area to prioritize for a quick start. Epics are created in the relevant development project.
+2. Plan user flows and interaction design. (Optional)
```

Comment:
Why is this step listed as optional? To make sure we plan content resources accordingly, the "Work with Abi to figure out who we work with in CCS and tag in JIRA story" should happen at the beginning of every quick start effort.

Reply:
The user flows and interaction design are marked optional since a lot of that design work has been done with the framework that is already in place. Future quick starts probably won't need as much, if any, design work from that perspective, requiring mostly content contributions. Reading the full description though, I see how that can be misleading. Should we rename this step to exclude interaction design and then remove the Optional label? Curious @beanh66 's thoughts since she was concerned with making it a mandatory step.

Reply:
@abigaeljamie I don't think that's the reality of the feature unfortunately. With this becoming extensible, we can't be blockers for each quick start that teams want to contribute. If you want, instead of (Optional), we could add a line at the top that says "Note: interaction design requirements may vary and some quick starts may not require wireframes." I think we will very much run into this and already are with the tech marketing team looking to contribute some of their workshop flows. In some ways it will be easier to try the PR directly in the console and work backwards to fix the flow rather than trying to design first and write second.

Reply:
Hi, @bmignano and @beanh66 ! I'm not referring to the interaction design piece. I was referring to the line reading, "Work with Abi to figure out who we work with in CCS and tag in JIRA story." That is part of #2, and because #2 is marked as optional, it looks as if we're saying everything under #2 is optional, including looping in content at the beginning. 

I think we should clarify what areas of #2 are optional (wireframes) and what parts aren't optional (including UXD Content at the beginning of the process). I just want to make sure that content isn't left out of the loop again 👍